### ISSUE TITLE
MIDI problem

### ISSUE THREAD

**Issue by hanbingniao1** [chegewara/EspTinyUSB#151]
Under Windows 11, MIDI can be recognized by the device manager, but MIDI messages are not seen in various DAW software and MIDI-OX

**Comment by test1025** [comment]
I have also tried using a mobile phone, which can recognize MIDI devices, but there is no MIDI message output

**Comment by hanbingniao1** [comment]
I have also tried using a mobile phone, which can recognize MIDI devices, but there is no MIDI message output

**Comment by erb997** [comment]
same thing on Win10
I can connect and show midi.product name of my choosing. midi.manufacturer doesn't show in device detail. GrandOrgue can see the device but doesn't show any events in log or play any notes. Did this ever get solved? I am using an ESP32-S3 board.

**Comment by hanbingniao1** [comment]
> same thing on Win10 I can connect and show midi.product name of my choosing. midi.manufacturer doesn't show in device detail. GrandOrgue can see the device but doesn't show any events in log or play any notes. Did this ever get solved? I am using an ESP32-S3 board.

No, I'm no succeeded ，I was in the same situation as you at that time……

**Comment by infrafast** [comment]
hello my device esp32-s3 (seed xiao) is also seen correectly as midi device under  windows  11 and macos but no notes are catched by any software. Any help appreciated on how you solved it.

**Comment by hanbingniao1** [comment]
> hello my device esp32-s3 (seed xiao) is also seen correectly as midi device under windows 11 and macos but no notes are catched by any software. Any help appreciated on how you solved it.

Actually, I haven't been successful yet.My problem phenomenon is completely the same as yours...TxT...

**Comment by infrafast** [comment]
> hello my device esp32-s3 (seed xiao) is also seen correectly as midi device under windows 11 and macos but no notes are catched by any software. Any help appreciated on how you solved it.

Actually I have added midi.setBaseEP(2); in the setup() and it now works for me

**Comment by hanbingniao1** [comment]
> > hello my device esp32-s3 (seed xiao) is also seen correectly as midi device under windows 11 and macos but no notes are catched by any software. Any help appreciated on how you solved it.
> 
> Actually I have added midi.setBaseEP(2); in the setup() and it now works for me

Wow, thank you for sharing! I will try it out later
